<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FGEObjectInteractionForm</class>
 <widget class="QWidget" name="FGEObjectInteractionForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>618</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QWidget" name="widget_menu_interaction" native="true">
     <property name="minimumSize">
      <size>
       <width>600</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>600</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">.QPushButton{
	border : 0px solid;
	height : 30px;
	background-color: rgb(225, 225, 225);
	color: rgb(84, 84, 84);
}
.QPushButton::hover{
	background-color: rgb(133, 28, 147);
	color: rgb(84, 84, 84);
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="pushButton_object_mode">
        <property name="text">
         <string>Object Mode</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_edit_mode">
        <property name="text">
         <string>Edit Mode</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_sculpt_mode">
        <property name="text">
         <string>Sculpt Mode</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_vertex_paint">
        <property name="text">
         <string>Vertex Paint</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_weight_paint">
        <property name="text">
         <string>Weight Paint</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_texture_paint">
        <property name="text">
         <string>Texture Paint</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="widget_slider" native="true">
     <property name="minimumSize">
      <size>
       <width>600</width>
       <height>300</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>600</width>
       <height>300</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
